src/hcb/artifacts/make_detection_fraction_plot.py

```diff
@@ -34,26 +34,29 @@ def plot_detection_fraction(all_data: MultiStats) -> plt.Figure:
     included_styles = {
         'honeycomb_SD6': ("SD6", "Periodic Honeycomb Code"),
         'honeycomb_SI1000': ("SI1000", "Periodic Honeycomb Code"),
-        "honeycomb_EM3_v2": ("EM3", "Periodic Honeycomb Code"),
+        "honeycomb_EM3_v3": ("EM3", "Periodic Honeycomb Code"),
```

Comment:
Does this actually work? I don't think we have v3 data for periodic (and I don't think there's a distinction w.r.t. v2 since it has no MPP-single-qubit cases)

Reply:
Ah I hadn't noticed that v3 idnt different for periodic, good point,
I more sort of assumed that we'd either rerun or at least re-name the data we had so it was obvious what error model everything corresponded too